base-template.sh

```diff
@@ -484,27 +483,73 @@ run_hook_file() {
 #   1 in case a hook fails, 0 otherwise
 #####################################################
 process_shared_hooks() {
-    SHARED_REPOS_LIST="$1"
-    shift
+    SHARED_HOOKS_TYPE="$1"
+    SHARED_REPOS_LIST="$2"
+    shift 2
```

Comment:
Is this POSIX?
If not (or unsure like I am) perhaps do
```
SHARED_HOOKS_TYPE="$1"
shift
SHARED_REPOS_LIST="$1"
shift
```

Reply:
Its Posix yes https://pubs.opengroup.org/onlinepubs/009695399/utilities/shift.html